examples/devices/lighty-network-topology-device/src/test/resources/expected_values.xml

```diff
@@ -0,0 +1,12 @@
+<values>
```

Comment:
really, this is not easy-readable
xml are structured format by reason, you need to use it according that - split expected values to separate files or assert specific values in tests